ferrocene/doc/internal-procedures/src/release/stable.rst

```diff
@@ -59,10 +59,61 @@ the latest commit on that branch into the ``beta-${major_version}`` channel
 every night. You can continue landing changes into the branch until you
 are ready to release it as a stable release.
 
+Add version to Known Problems
+-----------------------------
+
+Add the new version to the `Known Problems repository <https://github.com/ferrocene/problems/>`_
+by adding the version and branch to the ``src/config.yml`` file then running the tool.
+
+Validate that the site now has the version and known problems are tracked for it.
+
+Version Bump Release Notes
+--------------------------
+
+Rename ``ferrocene/doc/release-notes/src/next.rst`` to
+``ferrocene/doc/release-notes/src/${version}.rst``.
+Ensure the upcoming release is marked with ``:upcoming-release:``.
+
+Create a new ``ferrocene/doc/release-notes/src/next.rst`` with the following content:
+
+.. code-block::
+
+   .. SPDX-License-Identifier: MIT OR Apache-2.0
+      SPDX-FileCopyrightText: The Ferrocene Developers
+
+   :upcoming-release:
+
+   Next Ferrocene release
+   ======================
+
+   This page contains the changes to be introduced in the upcoming Ferrocene
+   release.
+
+Backport this PR according to :ref:`handling-backports`. Then, on the
+``release/1.NN`` branch, remove the ``:upcoming-release:`` from the version.
```

Comment:
hm, why are we backporting this... is it not that at this time, the release branch would already have the right file

Reply:
Good point!

Reply:
So I re-read the document, and there is no step saying to do the release note move or the `:upcoming-release:` removal prior to this, so this is the step that makes the release branch have the right file...